package.json

```diff
@@ -27,6 +27,7 @@
     "eslint-plugin-jsx-a11y": "6.7.1",
     "eslint-plugin-react": "7.32.2",
     "eslint-plugin-react-hooks": "4.6.0",
+    "sass-embedded": "^1.81.0",
```

Comment:
Should we use this package, or the sass one?

Reply:
We had a meeting earlier today, we talked about adding `vite` and `sass-embedded` to the top/root package.json.

Reply:
One that note, I ran into this problem with my my Apple M1 forcing me to delete `package-lock.json` https://github.com/npm/cli/issues/4828